srcpkgs/brother-brscan5/template

```diff
@@ -0,0 +1,73 @@
+# Template file for 'brother-brscan5'
+pkgname=brother-brscan5
+version=1.2.12
+revision=1
+archs="i686 x86_64"
+depends="sane libusb-compat"
+short_desc="SANE scanner driver for brscan5-compatible printers"
+maintainer="Alexandru Barbur <alex@ctrlc.name>"
+license="custom:EULA"
+homepage="https://support.brother.com/"
+repository="nonfree"
+nopie=yes
+conf_files="/etc/opt/brother/scanner/brscan5/brscan5.ini
+ /etc/opt/brother/scanner/brscan5/brsanenetdevice.cfg
+ /opt/brother/scanner/brscan5/models/*.ini"
+
+_sane_library="libsane-brother5.so.1.0.7"
+_pkg_libraries="libLxBsScanCoreApi.so.3.2.0
+ libLxBsNetDevAccs.so.1.0.0
+ libLxBsDeviceAccs.so.1.0.0
+ libLxBsUsbDevAccs.so.1.0.0"
+
+if [ "$XBPS_TARGET_MACHINE" = "x86_64" ]; then
+	_rpmpkgid="0.x86_64"
+	distfiles="https://download.brother.com/welcome/dlf104036/brscan5-${version}-${_rpmpkgid}.rpm"
+	checksum="b40ef093be58d298e4429d8c5d4b7ecb48db58b64a5a01c203c9325c14997b47"
+	_rpmlibdir="usr/lib64"
+else
+	_rpmpkgid="0.i386"
+	distfiles="https://download.brother.com/welcome/dlf104035/brscan5-${version}-${_rpmpkgid}.rpm"
+	checksum="7e98f36494e4145c9192067519fe8a0b105eb54ec9ba8ce6474bfc2882f07e67"
+	_rpmlibdir="usr/lib"
+fi
+
+do_install() {
```

Comment:
almost all of these comments are self-explanatory and not necessary

Reply:
Ok, removed.